contracts/Colony.sol

```diff
@@ -24,6 +24,8 @@ import "./EtherRouter.sol";
 
 contract Colony is ColonyStorage, PatriciaTreeProofs {
 
+  function() external payable {}
```

Comment:
Can we add a comment explaining what this function is for?

Reply:
This is no longer there.

Reply:
👍